Index: src/Resources/ContactManager.java
===================================================================
diff --git a/src/Resources/ContactManager.java b/src/Resources/Contacts/ContactManager.java
rename from src/Resources/ContactManager.java
rename to src/Resources/Contacts/ContactManager.java
--- a/src/Resources/ContactManager.java	(revision b2437b5a28ea828bc03884add55cfcfe1ce61aff)
+++ b/src/Resources/Contacts/ContactManager.java	(date 1773151772943)
@@ -1,5 +1,7 @@
 
-package Resources;
+package Resources.Contacts;
+
+import Resources.Data.UserData;
 
 import java.io.BufferedReader;
 import java.io.File;
Index: src/Resources/EmergencyManager.java
===================================================================
diff --git a/src/Resources/EmergencyManager.java b/src/Resources/emergency/EmergencyManager.java
rename from src/Resources/EmergencyManager.java
rename to src/Resources/emergency/EmergencyManager.java
--- a/src/Resources/EmergencyManager.java	(revision b2437b5a28ea828bc03884add55cfcfe1ce61aff)
+++ b/src/Resources/emergency/EmergencyManager.java	(date 1773151772937)
@@ -1,4 +1,7 @@
-package Resources;
+package Resources.emergency;
+import Resources.Data.JsonDataLoader;
+import Resources.Data.UserData;
+
 import java.util.List;
 
 public class EmergencyManager {
